proc_IR.py

```diff
@@ -168,24 +164,24 @@ def as_scan_nbr(d):
     fl.plot(s)
     x = s_signal.fromaxis('vd')
     f = fitdata(s_signal)
-    error = fitdata(s_forerror)
+    #error = fitdata(s_forerror)
     M0,Mi,R1,vd,W = symbols("M_0 M_inf R_1 vd W", real=True)
     if IR:
-        error.functional_form = Mi + (M0-Mi)*s_exp(-vd*R1)
+        #error.functional_form = Mi + (M0-Mi)*s_exp(-vd*R1)
         f.functional_form = Mi + (M0-Mi)*s_exp(-vd*R1)
     else:
-        error.functional_form = Mi*(1-(2-s_exp(-W*R1))*s_exp(-vd*R1))
+        #error.functional_form = Mi*(1-(2-s_exp(-W*R1))*s_exp(-vd*R1))
         f.functional_form = Mi*(1-(2-s_exp(-W*R1))*s_exp(-vd*R1))
-    error.fit()
+    #error.fit()
```

Comment:
I don't understand why we were fitting the error.  Just remove these lines that are commented out.